go/obscuronode/enclave/enclave.go

```diff
@@ -867,6 +882,26 @@ func (e *enclaveImpl) encryptTxReceiptWithViewingKey(address common.Address, txR
 	return e.rpcEncryptionManager.EncryptWithViewingKey(address, txReceiptBytes)
 }
 
+func (e *enclaveImpl) checkRollup(r *obscurocore.Rollup) {
```

Comment:
Can you add a comment saying what checks are happening? It's not clear to me based on the code.